### PR TITLE
VIH-9510 Witness Interpreter VMR cannot be called into hearing

### PR DESCRIPTION
**Pull request by oliver-scott**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-9510

### Change description ###
Fixes an issue where if a witness joins a conference, the conference is paused, and then an interpreter is added, the witness + interpreter cannot be called into the conference.

- The root cause is an error getting the witness room due to the witness not being present. After getting the room, refresh the cache to re-call the API if the witness wasn't found in the room

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/vh-video-web/values.dev.template.yaml [hunk 1]
```diff
@@ -3,11 +3,12 @@ java:
     image: '${IMAGE_NAME}'
     ingressHost: ${SERVICE_FQDN}
     releaseNameOverride: ${RELEASE_NAME}
-    disableTraefikTls: false
+    disableTraefikTls: true
```
Comment (oliver-scott) [inline charts/vh-video-web/values.dev.template.yaml:6]: TODO revert these changes before merge

**Review by will-craig — DISMISSED** [review]

**Review by komaldabhi1509 — DISMISSED** [review]

**Review by komaldabhi1509 — APPROVED** [review]

**Review by will-craig — APPROVED** [review]

**Review by komaldabhi1509 — APPROVED** [review]

**Review by komaldabhi1509 — APPROVED** [review]
